index.js

```diff
@@ -33,6 +39,22 @@ GzipFilter.prototype.processFile = function(srcDir, destDir, relativePath) {
   return Filter.prototype.processFile.apply(this, arguments);
 };
 
+GzipFilter.prototype.baseDir = function() {
+  return __dirname;
+}
+
+GzipFilter.prototype.optionsHash =  function() {
+  if (!this._optionsHash) {
+    this._optionsHash = stringify(this.options);
```

Comment:
Just using the JSON string here seems like it'll generate massive cache keys. It looks like broccoli-babel-transpiler [just MD5s it](https://github.com/babel/broccoli-babel-transpiler/blob/master/index.js#L157)—does that seem like a reasonable option to you?